render.c

```diff
@@ -79,10 +79,13 @@ static int render_notification(cairo_t *cairo, struct mako_state *state,
 	int notif_width =
 		(style->width <= state->width) ? style->width : state->width;
 
-	// Calculate the appropriate offset if we're right-aligned.
-	bool right_align =
-		(state->config.anchor & ZWLR_LAYER_SURFACE_V1_ANCHOR_RIGHT);
-	int offset_x = right_align ? (state->width - notif_width) : 0;
+	int offset_x;
+	if (state->config.anchor & ZWLR_LAYER_SURFACE_V1_ANCHOR_RIGHT) {
+		offset_x = state->width - notif_width - style->margin.right;
+	}
+	else {
```

Comment:
Style: `}` and `else {` on the same line

Reply:
The one style thing I can never adjust to :)